packages/vite/src/node/plugins/css.ts

```diff
@@ -120,6 +120,8 @@ export const chunkToEmittedCssFileMap = new WeakMap<
   Set<string>
 >()
 
+export const romovedPureCssFiles = new Map<string, RenderedChunk>()
```

Comment:
same here

Reply:
The `removedPureCssFiles` should behave like the `moduleCache`

https://github.com/vitejs/vite/blob/a03e94419ef4da3b842f0211536afb1501bc0919/packages/vite/src/node/plugins/css.ts#L149-L153

Reply:
Thank you for your suggestion! I've changed it. 😄